e2e_test/batch/types/jsonb.slt.part

```diff
@@ -150,3 +150,113 @@ SELECT '{
 		true}'::jsonb; -- OK
 ----
 {"one": 1, "three": true, "two": "two"}
+
+# jsonb_array_elements
+query T
+select * from jsonb_array_elements('[1,true, [2,false]]'::jsonb);
+----
+1
+true
+[2, false]
+
+statement error cannot extract elements
+select * from jsonb_array_elements('null'::jsonb)
+
+statement error cannot extract elements
+select * from jsonb_array_elements('1'::jsonb)
+
+statement error cannot extract elements
+select * from jsonb_array_elements('"string"'::jsonb)
+
+statement error cannot extract elements
+select * from jsonb_array_elements('{}'::jsonb)
+
+
+# jsonb_array_elements_text
+query T
+select * from jsonb_array_elements_text('["foo", "bar"]'::jsonb);
+----
+foo
+bar
+
+statement error cannot extract elements
+select * from jsonb_array_elements_text('null'::jsonb)
+
+statement error cannot extract elements
+select * from jsonb_array_elements_text('1'::jsonb)
+
+statement error cannot extract elements
+select * from jsonb_array_elements_text('"string"'::jsonb)
+
+statement error cannot extract elements
+select * from jsonb_array_elements_text('{}'::jsonb)
+
+# jsonb_object_keys
+query T
+select * from jsonb_object_keys('{"f1":"abc","f2":{"f3":"a", "f4":"b"}}'::jsonb);
+----
+f1
+f2
+
+statement error cannot call jsonb_object_keys
+select * from jsonb_object_keys('null'::jsonb)
+
+statement error cannot call jsonb_object_keys
+select * from jsonb_object_keys('1'::jsonb)
+
+statement error cannot call jsonb_object_keys
+select * from jsonb_object_keys('"string"'::jsonb)
+
+statement error cannot call jsonb_object_keys
+select * from jsonb_object_keys('[]'::jsonb)
+
+
+# jsonb_each
+query TT
+select * from jsonb_each('{"a":"foo", "b":"bar"}'::jsonb);
+----
+a "foo"
+b "bar"
+
+query T
+select jsonb_each('{"a":"foo", "b":"bar"}'::jsonb);
+----
+(a,"""foo""")
+(b,"""bar""")
```

Comment:
Found a problem in string formatting in struct. cc @xiangjinwu 

```
    [Diff] (-expected|+actual)
    -   (a,"""foo""")
    -   (b,"""bar""")
    +   (a,"foo")
    +   (b,"bar")
```

The quote `"` is escaped to `\"` in array, but `""` in record.

```
postgres=# select row(1, '"str"');
      row      
---------------
 (1,"""str""")
postgres=# select array['"str"'];
    array    
-------------
 {"\"str\""}
```

Is there any documents for this behavior? How should we handle it?

Reply:
Yes it is buggy right now: #4967

Reply:
> Yes it is buggy right now: #4967

Seems not related. Wrong issue number?

Reply:
My bad. Updated to #4769

Reply:
Thanks! Then I'd like to change the test result to fit the current format and leave this issue for the future.

Reply:
Yes, that works. Alternatively, we can write the tests without relying on the display of struct:
```
select (st).key, (st).value from (select jsonb_each('{"a":"foo", "b":"bar"}'::jsonb)) as t(st);
----
a "foo"
b "bar"
```